modules/admin_manual/pages/configuration/files/manual_ file_ locking.adoc

```diff
@@ -0,0 +1,34 @@
+= Manual File Locking
+:toc: right
+
+== Introduction
+
+Starting with ownCloud Server 10.5 a new enhancement is provided for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out).
+
+The feature builds on the WebDAV Locks backend which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
+
+== Installation
+
+By default locks set in the web interface will expire after 30 minutes. The maximum lock time by default is 24 hours.
```

Comment:
```suggestion
By default, locks set in the web interface will expire after 30 minutes. The maximum lock time by default is 24 hours.
```
Pls also make a note that this can be changed and make a reference to configuration below.

Reply:
You don’t need a comma in this. You can set it there but it’s not mandatory

Reply:
From the sentence pov it reads much better/easier. I take care on such things...
(You can try to speak this senctence with/without comma and you see the difference 😄 )